README.md

```
@@ -246,6 +252,44 @@ hooks:
 This behavior applies regardless of where you run `wtp add` from (main worktree
 or any other worktree).
 
+### Worktree Namespacing
+
+By default, wtp automatically namespaces worktrees by repository name to prevent
+conflicts when multiple projects share a parent directory:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)